### ISSUE TITLE
Server browser UI overlap on controller

### ISSUE THREAD

**Issue by Jan200101** [R2Northstar/NorthstarMods#738]
![grafik](https://github.com/R2Northstar/NorthstarMods/assets/15076013/31be68e1-3bf0-4b90-8a30-5c3056a3f725)
![grafik](https://github.com/R2Northstar/NorthstarMods/assets/15076013/48030dc4-e40c-44c8-82a2-49b1cbab1e3a)

**Comment by ASpoonPlaysGames** [comment]
Yeah, this happens because we don't use footer buttons for the player/server count and someone manually lined them up with the button (and forgot about controller)
